### PR TITLE
TELCODOCS-628: Disabling chronyd in PCG CR

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.10

Issue: https://issues.redhat.com/browse/TELCODOCS-628

Link to docs preview: http://file.emea.redhat.com/amolnar/TELCODOCS-628-RN-4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-RAN-disabling-chronyd

Additional information:

### CODE REVIEW

**Review by tliu2021 — COMMENTED** [review]
The addition should be included to the overlaid spec file of TunedPerformancePatch.yaml (vs source-crs), which is normally included in the group policies.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -1165,15 +1165,16 @@ For more information, see xref:../scalability_and_performance/cnf-numa-aware-sch
 [id="ocp-4-10-RAN-disabling-chronyd"]
 ====  Disable chronyd in the PolicyGenTemplate CR
 
-You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the group `PolicyGenTemplate` CR. Add the following line in `TunedPerformancePatch.yaml`:
+You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the `source-crs` directory of the `ztp-site-generate` container image:
```
Comment (tliu2021) [inline release_notes/ocp-4-10-release-notes.adoc:1168]: This is referring the PolicyGenTemplate CR in the group policies (not the source-crs)
 - fileName: TunedPerformancePatch.yaml
      policyName: "config-policy"
      spec:
        profile:
          - name: performance-patch
            # The 'include' line must match the PerformanceProfile metadata.name above (openshift-node-performance-${metadata.name})
            # And the cmdline_crash CPU set must match the 'isolated' set in the PerformanceProfile above
            data: |
              [main]
              summary=Configuration changes profile inherited from performance created tuned
              include=openshift-node-performance-openshift-node-performance-profile
              [bootloader]
              cmdline_crash=nohz_full=2-23,26-47
              [sysctl]
              kernel.timer_migration=1
              [service]
              service.stalld=start,enable
              service.chronyd=stop,disable

Comment (abhatt-rh) [reply release_notes/ocp-4-10-release-notes.adoc]: @tliu2021 Thank you for the clarification. 
Sorry that it was not immediately clear to me where the [line](https://github.com/openshift/openshift-docs/pull/49033/files#diff-a5bd860487970afb9da12221d481f6220b28719d13763193710cd76ae315d2a2R1177) was required to be added. If I understand it correctly, you are referring to the group PolicyGenTemplates (or the `group` YAMLs) that can be found at argocd/example/policygentemplates in the `ztp` container image.

Comment (tliu2021) [reply release_notes/ocp-4-10-release-notes.adoc]: Yes, the example can be found at argocd/example/policygentemplates. The customers would need to update their own PolicyGenTemplate CR in their git repository.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1166,6 +1166,19 @@ For more information, see xref:../scalability_and_performance/cnf-numa-aware-sch
 ==== Filtering custom resources during ZTP spoke cluster installation using SiteConfig filters
 You can now use filters to customize `SiteConfig` CRs to include or exclude other CRs for use in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline. For more information, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-filtering-ai-crs-using-siteconfig_ztp-deploying-disconnected[Filtering custom resources using SiteConfig filters].
 
+[id="ocp-4-10-RAN-disabling-chronyd"]
+====  Disable chronyd in the PolicyGenTemplate CR
+
+You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the group `PolicyGenTemplate` CR:
```
Comment (abhatt-rh) [inline release_notes/ocp-4-10-release-notes.adoc:1172]: Just a minor nit based on @tliu2021's inputs, would the following be more appropriate?
```suggestion
You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is referenced in the group `PolicyGenTemplate` CR:
```

**Review by opayne1 — APPROVED** [review]

**Pull request merged**